docs/rfcs/XXX-pull-request-review.md

```diff
@@ -0,0 +1,145 @@
+
+Clicking the `+` reveals a new comment box, which may be used to submit a single comment or begin a multi-comment review:
+
+![single-review](https://user-images.githubusercontent.com/378023/40351475-78a527c2-5de7-11e8-8006-72d859514ecc.png)
+
+## Drawbacks
+
+This adds a substantial amount of complexity to the UI, which is only justified for users that use GitHub pull request reviews.
+
+Showing all reviews in the current pull request tile can easily overwhelm the other pull request information included there. It also limits our ability to expand the information we provide there in the future (like associated issues, say).
+
+Rendering pull request comments within TextEditors can be intrusive: if there are many, or if your reviewers are particularly verbose, they could easily crowd out the code that you're trying to write and obscure your context.
+
+## Rationale and alternatives
+
+One alternative may be to show review comments _only_ within the "changes" tab of an `IssueishPaneItem`. This simplifies flow considerably, because it removes the need to provide navigation among different views of the same review, and unifies the handling of current and non-current pull requests. However, I believe that the renderings of reviews in all three places each serve a unique purpose:
+
+* Reviews within the "Changes" tab of the `IssueishPaneItem` reveal a narrative formed by all of the reviews on a specific pull request together, as a conversation among reviewers.
+* Reviews within the "Review" tab of the `IssueishPaneItem` reveal the narrative flow within each review individually. For example, review comments that refer to other comments within the same review (e.g. "same here" or "and again") become clearer here.
+* Review comments within open TextEditors allow the reader to use more context within the source code to evaluate, address, or respond to each individual comment thread: consistency with functions that are not visible within the immediate diff, context within algorithms that span many lines. They also allow the receiver of a review to preserve the mental context of the review communication as they move back and forth between reading the content of a review and applying it to their source.
+
+## Unresolved questions
+
+### Questions I expect to address before this is merged
+
+Can we access "draft" reviews from the GitHub API, to unify them between Atom and GitHub?
+
+How do we represent the resolution of a comment thread? Where can we reveal this progress through each review, and of all required reviews?
+
+Are there any design choices we can make to lessen the emotional weight of a "requests changes" review? Peer review has the most value when it discovers issues for the pull request author to address, but accepting criticism is a vulnerable moment.
+
+Similarly, are there any ways we can encourage empathy within the review authoring process? Can we encourage reviewers to make positive comments or demonstrate humility and open-mindedness?
+
```

Comment:
Another idea for promoting constructive collaboration -- implement a quick and easy way to open and share a teletype portal so that folks can easily get input from teammates throughout the review process. Some of the back-and-forth in comments can be entirely side-stepped by a quick teletype/pairing session. 

Along these lines, it'd be cool to offer comment links that people can drop in Slack that opens the comment in Atom, or on dotcom for users who don't use Atom. I do this a lot with comments on dotcom, and I would want a similar affordance in Atom.